### ISSUE TITLE
Empty manifest produced when Maven plugin configuration is inherited

### ISSUE THREAD

**Issue by io7m** [bndtools/bndtools#1931]
Hello!

It seems that Bndtools produces an empty manifest when loading a Maven-based project with an invocation of the `bnd-maven-plugin` that uses a configuration inherited from a parent POM.

Here's a repro case:

https://github.com/io7m/osgitest3-20180902

When using `mvn clean package`, the resulting bundle manifest is quite extensive. However, when deleting the bundle and using "build" in Eclipse, the bundle manifest is almost empty:

```
  Manifest-Version: 1.0
  Created-By: Apache Maven
  Built-By: rm
  Build-Jdk: 10.0.2
```

Versions:

```
$ java -version
openjdk version "10.0.2" 2018-07-17
OpenJDK Runtime Environment (build 10.0.2+13)
OpenJDK 64-Bit Server VM (build 10.0.2+13, mixed mode)

Eclipse:
Version: Photon Release (4.8.0)
Build id: 20180619-1200

Bndtools	4.0.0.REL-20180511-182614-g64633ee	bndtools.main.feature.feature.group	Bndtools
Bndtools m2e	4.0.0.REL-20180511-182614-g64633ee	bndtools.m2e.feature.feature.group	Bndtools
```

**Comment by io7m** [comment]
Is there any chance that this issue will be looked at relatively soon?

I'm by no means pushing anyone to do work for me, but this issue is essentially blocking a migration to Eclipse from IDEA and so if this isn't likely to get looked at any time soon, I'll postpone the migration and do something else instead.

**Comment by bjhargrave** [comment]
The test case has zero Bnd configuration in the POM. Furthermore, you must configure the maven-jar-plugin to use to manifest made by bnd-maven-plugin:  https://github.com/bndtools/bnd/tree/master/maven/bnd-maven-plugin#important-note

**Comment by io7m** [comment]
'Ello.

The configuration is inherited from the parent. This is a common setup (the "organization-wide POM"). Take a look at the effective POM (`mvn help:effective-pom`).

**Comment by bjhargrave** [comment]
I could built it from mvn. I guess we need someone  who knows the Bndtools m2e integration to investigate. @gamerson ?

**Comment by io7m** [comment]
Yep, it works fine building on the command line (as mentioned), but Eclipse evidently sees nothing.
